app/Models/Celeb.php

```diff
@@ -8,4 +8,8 @@
 class Celeb extends Model
 {
     use HasFactory;
+
+    public function movies() {
+        return $this->belongsToMany('App\Movie');
```

Comment:
You can actually use the class instead of the string here Jayd ;D 

`return $this->belongsToMany(Movie::class);
`

Reply:
Thank you for that Vickk!!